docs/running-jobs/job-and-queue-scheduling.md

```diff
@@ -1,55 +1,128 @@
 `chunk`: *A set of resources allocated as a unit to a job. Specified inside a selection directive. All parts of a chunk come from the same host.  In a typical MPI (Message-Passing Interface) job, there is one chunk per MPI process.*
 
 `vnode`: *A virtual node, or vnode, is an abstract object representing a host or a set of resources which form a usable part of an execution host. This could be an entire host,
 or a nodeboard or a blade. A single host can be made up of multiple vnodes. Each vnode can be managed and scheduled independently. Each vnode in a complex must have a unique name. Vnodes on a host can share resources, such as node-locked licenses.*  PBS operates on vnodes.  A vnode can, and in ALCF often will, represent an entire host, but it doesn't have to.  For instance, there is a mode on Polaris where we could have each physical host look like four vnodes, each with 16 threads, 1/4 of the RAM and one A100.
 
-`ncpus`: In ALCF, given the way we configure PBS, this equates to a hardware thread.  For example, Polaris has a single socket with a 32 core CPU, each with two threads, so PBS reports that as ncpus=64.
+`ncpus`: Number of resources available to execute a program. In ALCF, given the way we configure PBS, this equates to a hardware thread.  For example, a single socket node with a 32 core CPU, each with two hardware threads would report that as ncpus=64.
 
-`ngpus`: The number of GPUs.  On Polaris, this will generally be four.  However, if we enable *Multi Instance GPU (MIG)* mode and use cgroups it could be as high as 28.
+`ngpus`: The number of allocable GPUs on the vnode.  For an Nvidia A100, this could be one, however, if we enable *Multi Instance GPU (MIG)* mode and use cgroups it could be as high as 7.
 
-### The basics
-If you are an existing ALCF user and are familiar with Cobalt, you will find the PBS commands very similar though the options to qsub are quite different.  Here are the "Big Four" commands you will use:
+## <a name="Quick-Start"></a>Quick Start
+If you are an ALCF user and are familiar with Cobalt, you will find the PBS commands very similar though the options to qsub are quite different.  Here are the "Big Four" commands you will use:
 
-1. `qsub` - request resources (compute nodes) to run your job and start your script/executable on the head node.
-2. `qstat` - check on the status of your request
+1. `qsub` - request resources (generally compute nodes) to run your job and start your script/executable on the head node.  Here is the minimal qsub allowed at the ALCF:
+    * `qsub -A <project> -l select=<# of nodes>,walltime=HH:MM:SS,filesystems=fs1:fs2 <your job script>`
+    * The `-A`, `walltime`, and `filesystems` are mandatory.  You will receive errors if they are not specified.
+    * We automatically add `-k doe` for you, but specifying it explicitly probably wouldn't hurt (streams your output back rather than spooling it and copying it back at the end of the job)
```

Comment:
Should we also mention that -k doe is not only set, but enforced?  That the user cannot modify it or else they will get errors on their submission?

Reply:
addressed in c9d2a7cc11f2efcece42ffbf7ed4576d15e635c5